### ISSUE TITLE
WUD exits on pushover failure

### ISSUE THREAD

**Issue by jasongordon** [fmartinou/whats-up-docker#26]
```
{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"What's up, docker? is starting","time":"2021-01-23T20:03:20.947Z","v":0}

{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"Init DB (/store/wud.json)","time":"2021-01-23T20:03:20.948Z","v":0}

{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"Init Prometheus module","time":"2021-01-23T20:03:20.958Z","v":0}

{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"Register all components for provider pushover","time":"2021-01-23T20:03:21.027Z","v":0}

{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"Register component 1 of type pushover with configuration {\"user\":\"\\\"******************************\\\"\",\"token\":\"\\\"******************************\\\"\",\"sound\":\"pushover\",\"priority\":0}","time":"2021-01-23T20:03:21.040Z","v":0}

{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"No Registry configured => Init a default one (Docker Hub with default options)","time":"2021-01-23T20:03:21.040Z","v":0}

{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"Register component hub of type hub with configuration {}","time":"2021-01-23T20:03:21.343Z","v":0}

{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"Register component local of type docker with configuration {\"watchbydefault\":false,\"socket\":\"/var/run/docker.sock\",\"port\":2375,\"cron\":\"0 * * * *\"}","time":"2021-01-23T20:03:21.842Z","v":0}

{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"Schedule runner (0 * * * *)","time":"2021-01-23T20:03:21.843Z","v":0}

{"name":"whats-up-docker","hostname":"88fbfcf23d72","pid":1,"level":30,"msg":"HTTP API exposed on port 3000","time":"2021-01-23T20:03:21.870Z","v":0}

/home/node/app/node_modules/pushover-notifications/lib/pushover.js:146

throw new Error(d.errors[0], res)

^


Error: application token is invalid

at Pushover.errors (/home/node/app/node_modules/pushover-notifications/lib/pushover.js:146:13)

at IncomingMessage.<anonymous> (/home/node/app/node_modules/pushover-notifications/lib/pushover.js:250:12)

at IncomingMessage.emit (events.js:327:22)

at endReadableNT (internal/streams/readable.js:1327:12)

at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

Here is my docker-compose.yml
```
version: '3'
services:
  wud:
    image: fmartinou/whats-up-docker:3.2.1
    ports:
        - 3001:3000
    volumes:
        - /Users/jgordon/docker/volumes/whats-up-docker:/store
        - /var/run/docker.sock:/var/run/docker.sock
    environment:
#        - WUD_TRIGGER_IFTTT_IFTTT1_KEY=XXXSNIPPEDXXXX
        - WUD_TRIGGER_PUSHOVER_1_TOKEN="abcdefgSNIPPED"
        - WUD_TRIGGER_PUSHOVER_1_USER="xyz123SNIPPED"
        - WUD_WATCHER_LOCAL_WATCHBYDEFAULT=FALSE
    labels:
       - 'wud.watch=true'
       - 'wud.tag.include=^[0-9]\d*.[0-9]\d*.[0-9]\d*$$'
```

**Comment by jasongordon** [comment]
I see you committed a fix for this.  Still can't figure out why my application ID is coming back as invalid though.  I use pushover with many other apps just fine.  Should they keys not be quoted in my docker-config.yml file?

user page (user field):
![pushover1](https://user-images.githubusercontent.com/2027525/105701008-abe7b200-5ed7-11eb-86af-53c0612778a5.PNG)

app page (token field):
![pushover2](https://user-images.githubusercontent.com/2027525/105701016-af7b3900-5ed7-11eb-86ed-610398922be9.PNG)

**Comment by fmartinou** [comment]
Yes, you should omit the quotes.

Actually, Docker-Compose supports multiple syntaxes:

``` yaml
services:
  wud:
    image: fmartinou/whats-up-docker:3.2.1

    # List of String => values should not be enclosed with quotes because quotes are part of the value
    environment:
        - WUD_TRIGGER_PUSHOVER_1_TOKEN=abcdefgSNIPPED
        - WUD_TRIGGER_PUSHOVER_1_USER=xyz123SNIPPED
```

``` yaml
services:
  wud:
    image: fmartinou/whats-up-docker:3.2.1

    # Map -> Values don't need to be enclosed with quotes (except if values contain special characters)
    environment:
        WUD_TRIGGER_PUSHOVER_1_TOKEN: abcdefgSNIPPED
        WUD_TRIGGER_PUSHOVER_1_USER: xyz123SNIPPED
```

``` yaml
services:
  wud:
    image: fmartinou/whats-up-docker:3.2.1

    # Map -> But if you enclose them with quotes, it still works
    environment:
        WUD_TRIGGER_PUSHOVER_1_TOKEN: abcdefgSNIPPED
        WUD_TRIGGER_PUSHOVER_1_USER: xyz123SNIPPED
```

From the UI, on the `trigger` page, you should see the real applied configuration for pushover (including your tokens masked with `*`)
![image](https://user-images.githubusercontent.com/16061231/105708929-5a5e1800-5f15-11eb-8383-0774ef6be196.png)

You can check there if the values seem ok.

**Comment by fmartinou** [comment]
it's fixed in version `3.2.2`; WUD won't stop if a Pushover UserId and/or ApiToken is invalid.
